acquia_lift.routing.yml

```diff
@@ -4,4 +4,47 @@ acquia_lift.admin_settings_form:
     _form: '\Drupal\acquia_lift\Form\AdminSettingsForm'
     _title: 'Acquia Lift settings'
   requirements:
-    _permission: 'administer acquia lift'
+    _permission: 'administer acquia_lift'
+
+acquia_lift.slot.overview:
+  path: '/admin/config/content/acquia-lift/slots'
```

Comment:
same here, the entity_type_id = acquia_lift_slot so we need to keep consistency here. Also, If we make this overview, /admin/config/content/acquia-lift/slots won't go anywhere unless we allow both paths to route to the same thing. Also fine by me, but not having /admin/config/content/acquia-lift/slots and having /admin/config/content/acquia-lift/slots/overview is also not acceptable. Thoughts?